### PR TITLE
e4s ci: add tasmanian +rocm

### PR DESCRIPTION
**Pull request by eugeneswalker**
FYI @mkstoyanov @wspear

### CODE REVIEW

**Review by tldahlgren — DISMISSED** [review]
LGTM

**Review by tldahlgren — COMMENTED** [review]
Looks like there are issues building the packages with this option.  Has it been reported?

**Comment by mkstoyanov** [comment]
I see the problem, will work on a solution.

**Comment by mkstoyanov** [comment]
By default, Tasmanian tries to use OpenMP for multithreaded work on the CPU side only. In some recent update, andgpu-llvm hid the OpenMP support behind a variant and I cannot even build the variant due to some conflict with offloading (which Tasmanian does not need).

I guess we would have to disable OpenMP.

File: share/spack/gitlab/cloud_pipelines/stacks/e4s/spack.yaml [hunk 1]
```diff
@@ -197,6 +197,7 @@ spack:
   - slepc +rocm amdgpu_target=gfx90a ^petsc +rocm amdgpu_target=gfx90a
   - strumpack ~slate +rocm amdgpu_target=gfx90a
   - superlu-dist +rocm amdgpu_target=gfx90a
+  - tasmanian +rocm amdgpu_target=gfx90a
```
Comment (mkstoyanov) [inline share/spack/gitlab/cloud_pipelines/stacks/e4s/spack.yaml]: This should resolve the problem.
```
tasmanian +rocm ~openmp amdgpu_target=gfx90a
```

Comment (eugeneswalker) [reply share/spack/gitlab/cloud_pipelines/stacks/e4s/spack.yaml]: Thank you! I've committed this change.

**Review by mkstoyanov — APPROVED** [review]

**Review by tldahlgren — APPROVED** [review]
LGTM

**Pull request merged**